dockerfiles/Dockerfile.ubuntu

```diff
@@ -1,3 +1,13 @@
+# Stage 0: install Node dependencies
+FROM node:18-alpine AS npm-installer
+
+WORKDIR /usr/src/app
+COPY package.json ./
+
+# install node.js dependencies e.g. Vue (but not the development dependencies)
+RUN npm install --omit=dev
```

Comment:
I checked using [can-i-ignore-scripts](https://www.npmjs.com/package/can-i-ignore-scripts) and it doesn't look like our non-dev packages need any install scripts, so I added the --ignore-scripts option. Since we have to depend on npm packages anyway and this option should only affect the throwaway npm-installer stage where the `npm install` command is executed, this doesn't look like a major security problem. But at least it should make SonarCloud happy.